flamegraph/flamegraph.py

```diff
@@ -18,21 +19,20 @@ def get_thread_name(ident):
 def default_format_entry(threadname, fname, line, fun, fmt='%(threadname)s`%(fun)s'):
   return fmt % locals()
 
-def create_flamegraph_entry(thread_id, frame, format_entry, collapse_recursion=False):
+def create_flamegraph_entry(thread_id, stack, format_entry, collapse_recursion=False):
```

Comment:
**Note to reviewers**: accepting a stack parsed by the caller makes this function easier to unit test.